docs/advanced/worker-configuration-reference.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: "Worker Config Reference"
+title: "Worker config reference"
 layout: default
 sort: 4
 ---
 
-# NFD-Worker Configuration File Reference
+# NFD-Worker configuration file reference
```

Comment:
Here as well...

Reply:
Changed to `Configuration file reference of nfd-worker`